src/components/accordion/Accordion.styles.ts

```diff
@@ -37,7 +37,9 @@ export const styles = {
       '& h6': {
         color: 'basic.white'
       }
-    }
+    },
+    summary: {},
+    details: {}
```

Comment:
If you don't use these styles, delete them

Reply:
they are needed for TS

Reply:
so do them not required in TS

Reply:
@MickhaelSh I have tried by removing summary from styles and making condition with &&, where sx={accordionStyle.summary && accordionStyle.summary} but TS still need to have these fields

Reply:
These fields should be optional.